### DISCUSSION TITLE
set conceallevel=0 doesn't work using extras.lang.tex?

### DISCUSSION THREAD

**Discussion by Zeta611** [LazyVim/LazyVim#2871] [Q&A] [answered]
Hi, I'm using `extras.lang.tex`, so that I can use vimtex and other goodies in (La)TeX documents.

However, I can't seem to `set conceallevel=0` except explicitly inside a buffer.
It doesn't work when I put `vim.o.conceallevel = 0` in `after/ftplugin/tex.lua` or `lua/config/options.lua`.

When I try to debug using `verbose set conceallevel?`, it says `conceallevel=2 Last set from Lua` which is not super helpful.

I guess something is setting `conceallevel=2` after my options are loaded; how can I find out which Lua script did this, and what should I do to force disable conceal?

Thanks!

**Comment by folke** [comment, marked as answer]
Not sure why that was added, but the extra for latex overrode the conceallevel. Just removed that code

https://github.com/LazyVim/LazyVim/commit/914f26d04e6b9a61f7d4fc80ad433d75dac3f14b

**Reply by Zeta611** [reply]
Thanks a lot!
